rio_tiler/models.py

```diff
@@ -298,7 +305,56 @@ def __iter__(self):
             yield i
 
     @classmethod
-    def create_from_list(cls, data: Sequence["ImageData"]):
+    def from_array(cls, data: numpy.ndarray) -> "ImageData":
+        """Create ImageData from a numpy array.
+
+        Args:
+            data (numpy.ndarray): Numpy array or Numpy masked array.
+
+        """
+        if len(data.shape) < 3:
+            data = numpy.expand_dims(data, axis=0)
+
+        if isinstance(data, numpy.ma.MaskedArray):
+            data = data.data
+            mask = ~numpy.logical_or.reduce(numpy.ma.getmaskarray(data)) * numpy.uint8(
+                255
+            )
+            return cls(data, mask)
+
+        return cls(data)
```

Comment:
This method can be useful when creating ImageData from Masked Array (not really when using regular numpy ndarray 😄)